source/_lovelace/glance.markdown

```diff
@@ -45,6 +45,11 @@ column_width:
   description: "Column width as CSS length like `100px` or `calc(100% / 7)`. This controls how many entities appear in a row - at the default 20% you have 5 entities in a row. Use `calc(100% / 7)` for 7 entities in a row, and so on."
   type: string
   default: 20%
+theming:
+  required: false
+  description: "Set to `primary` to style the card with the background and text color of the header bar."
+  type: string
+  default: "default"
```

Comment:
Judging by the code, there is no default explicitly set.
So I guess this line should be left out?